### ISSUE TITLE
Conversion tracking (condition of Google grant)

### ISSUE THREAD

**Issue by MyfanwyNixon** [mysociety/writetothem#179]
It is a new requirement for our Google Adwords grant that:

*Recipients must have installed Conversion Tracking and be successfully tracking at least one conversion. The conversion cannot simply be a page view of your homepage, but be tracking a substantial action, such as a newsletter or volunteer sign-up.*

Our Adwords currently point at WriteToThem.

I do not think we have conversion tracking in place for Adwords (though we do for Analytics) - instructions are [here](https://support.google.com/adwords/answer/1722021), and I do not see any WTT source code containing the indicated snippet.

Possibly this is something we looked at before and did not implement because of our privacy policy?

There are some figures in our Adwords dashboard which are titled 'converted clicks', but they are either 0 or very close to it.

Looking at Analytics, the [conversion rate from Adwords](https://www.google.com/url?q=https%3A%2F%2Fwww.google.com%2Fanalytics%2Fweb%2F%23report%2Ftrafficsources-campaigns%2Fa660910w56920593p58054409%2F) is 3.73%, which, while low, is still better than the 0.0% indicated in the Adwords interface. My conclusion is that we do not have conversion tracking enabled in Adwords and that if we do, it is erroring in some way.

In short, if we want to retain our grant, we need to ensure that this is set up in Adwords (and change our privacy policy if necessary?).

**Comment by MyfanwyNixon** [comment]
Duplicate of #301
